src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -522,16 +522,15 @@ private Plan.KeysIteration buildInequalityPlan(Expression predicate)
      * @param builder The plan node builder which receives the built index scans
      * @param expressions The expressions to build the plan from
      */
-    public void buildPlanForExpressions(Plan.Builder builder, Collection<Expression> expressions)
+    protected void buildPlanForExpressions(Plan.Builder builder, Collection<Expression> expressions)
```

Comment:
Can use default visibility then? There are no subclasses, so you don't need protected.